boards/arm/b_u585i_iot02a/doc/index.rst

```diff
@@ -346,7 +346,7 @@ can be generated using ``b_u585i_iot02a_ns`` as build target.
 
    $ west build -b b_u585i_iot02a_ns path/to/source/directory
 
-Note: When building the ``*_ns`` image with TF-M, ``build/tfm/postbuild.sh`` bash script
+Note: When building the ``*_ns`` image with TF-M, ``build/tfm/api_ns/postbuild.sh`` bash script
 is run automatically in a post-build step to make some required flash layout changes.
 
 Once the build is completed, run the following script to initialize the option bytes.
```

Comment:
To be fully complete, you'd need to update the pre-flashing instruction as well:
```
-$ ./build/tfm/regression.sh
+$./build/tfm/api_ns/regression.sh
```

Reply:
good point, changed it